modules/telestion-application/src/main/java/de/wuespace/telestion/application/launcher/ConfigDeploymentLauncher.java

```diff
@@ -0,0 +1,9 @@
+package de.wuespace.telestion.application.launcher;
+
+/**
+ * This launcher combines the functionality of a {@link ConfigLauncher} and a {@link DeploymentLauncher}.
```

Comment:
I think we should be careful with the terminology here. Maybe call it "launcher type" or something like this. "Launcher", to me, sounds like a concrete launcher, not an interface.

Reply:
Done in [`2ed8921` (#501)](https://github.com/wuespace/telestion-core/pull/501/commits/2ed89215e19e84a289291393130591d3ae5e23d9)